Azure/sonic-buildimage
# Add switch ASIC vendor and platform for Nephos

**- What I did**
    Add switch ASIC vendor: Nephos
    Add Nephos platform: Ingrasys S9130-32X

**- How I did it**
    Add platform/nephos files
    Add platform/nephos/sonic-platform-modules-ingrasys submodule
    Add device/ingrasys/x86_64-ingrasys_s9130_32x-r0 files
    Add SONiC to support Nephos platform

**- How to verify it**
    To build SONiC installer image and docker images, run the following commands:
       make configure PLATFORM=nephos
       make target/sonic-nephos.bin
    Check system and network feature is worked as well

**- Description for the changelog**
    Add switch ASIC vendor and platform for Nephos

**- A picture of a cute animal (not mandatory but encouraged)**

Signed-off-by: Sam Yang <yang.kaiyu@gmail.com>
